modules/300-prometheus/templates/grafana/ingress-v10.yaml

```diff
@@ -32,19 +35,19 @@ metadata:
     # This rewrite forbids users to log in and get higher privileges.
     nginx.ingress.kubernetes.io/server-snippet: |
       location = /login {
-        rewrite ^ {{ include "helm_lib_module_uri_scheme" . }}://{{ include "helm_lib_module_public_domain" (list . "grafana-v10") }} permanent;
+        rewrite ^ {{ include "helm_lib_module_uri_scheme" . }}://{{ include "helm_lib_module_public_domain" (list . "grafana") }} permanent;
       }
     nginx.ingress.kubernetes.io/affinity: cookie
 spec:
   ingressClassName: {{ include "helm_lib_module_ingress_class" . | quote }}
   {{- if (include "helm_lib_module_https_ingress_tls_enabled" .) }}
   tls:
   - hosts:
-    - {{ include "helm_lib_module_public_domain" (list . "grafana-v10") }}
+    - {{ include "helm_lib_module_public_domain" (list . "grafana") }}
```

Comment:
Need to add `- {{ include "helm_lib_module_public_domain" (list . "prometheus") }}` and remove it from ingress.yaml.